source/manual/development-pipeline.html.md

```diff
@@ -49,12 +44,28 @@ Sometimes you may need to deploy your change in Integration in order to test it
 - `TAG` - put the name of your branch
 - Typically you can leave the checkboxes as they are
 
+## Check if there is a deploy freeze
+
+In exceptional circumstances, we may wish to block or _freeze_ deployments for a short period of time. This should be done by checking "Freeze deployments?" and adding an explanatory note to the application in the [Release app][release].
+
+> Checking "Freeze deployments?" will turn off all automatic deployments for the application. You can still deploy urgent changes manually if necessary.
+
+When a deploy freeze is in effect, you should avoid merging any PRs. This is because your changes may block other, urgent changes related to the deploy freeze. Your changes will also remain undeployed for a long time.
+
+> People don't always check the Release app before merging their PRs. If you need to freeze deployments for an application, you should also make people aware using other channels.
```

Comment:
Yup - this all makes sense! 

Side thought and (probably out of scope) of this PR:
On line 55, the docs are telling people that they should check deploy freezes, then acknowledges that people don't always check. Which probably leaves the reader thinking - "do I actually need to do this?". I don't think the answer to this is with how the documentation is written, and probably highlights the ambiguity in the "deploy freeze" process.

Reply:
I know what you mean! We focus a lot on freezing the deployment, but not freezing merging; people do usually mention this in comms, though. This should a least be enough to encourage someone to send out comms as part of a freeze.